metpy/plots/tests/test_skewt.py

```diff
@@ -50,6 +50,28 @@ def test_skewt_api():
     return fig
 
 
+@pytest.mark.mpl_image_compare(tolerance={'2': 9.57}.get(MPL_VERSION, 0.027),
+                               remove_text=True, style='default')
+def test_skewt_api_units():
+    """#Test the SkewT API when units are provided."""
+    with matplotlib.rc_context({'axes.autolimit_mode': 'data'}):
+        fig = plt.figure(figsize=(9, 9))
+        skew = SkewT(fig)
+        p = np.linspace(950, 100, 10) * units.hPa
+        t = np.linspace(18, -20, 10) * units.degC
```

Comment:
Can you change this to Kelvin (with the same range in C)? That would help make this test the use case around some of the other issues we've had.